modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -13,6 +13,406 @@
+** .heif or .heic (High Efficiency Image File Format)
+** .ico (Icon File)
+** .tga (Targa Graphic)
+** .pcx (Paintbrush Bitmap Image)
+** .ai (Adobe Illustrator File)
+** .eps (Encapsulated PostScript)
+** .wmf (Windows Metafile)
+** .emf (Enhanced Metafile)
+** .xcf (eXperimental Computing Facility, native GIMP file format)
+** .indd (Adobe InDesign)
+** .cr2 (Canon Raw 2nd Edition)
+** .nef (Nikon Electronic Format)
+** .orf (Olympus Raw Format)
+** .sr2 (Sony Raw Format 2)
+** .pef (Pentax Electronic File)
+** .arw (Sony Alpha Raw)
+** .rw2 (Panasonic RAW 2)
+** .dng (Digital Negative)
+** .exr (OpenEXR)
+** .apng (Animated Portable Network Graphics)
+** .avif (AV1 Image File Format)
+** .jxr (JPEG XR)
+** .hdp (HD Photo, also known as JPEG XR)
+** .cpt (Corel Photo-Paint image)
+** .dds (DirectDraw Surface)
+** .jp2 or .j2k (JPEG 2000)
+** .jng (JPEG Network Graphics)
+** .pbm (Portable Bitmap Format)
+** .pgm (Portable Graymap Format)
+** .ppm (Portable Pixmap Format)
+** .pnm (Portable Any Map)
+** .pfm (Portable Float Map)
+** .pam (Pluggable Authentication Module)
+** .hdr (High Dynamic Range Image)
+** .ras (Sun Raster Graphic)
+** .sgi or .rgb (Silicon Graphics Image)
+** .tiff (Tagged Image File Format, alternative spelling)
+** .xbm (X Bitmap)
+** .xpm (X Pixmap)
+** .ico (Windows Icon)
+** .cur (Windows Cursor)
+
+// do not remove
+
+[none]
+** ... and all other types that are included within the MIME Types of image/*
+
+// do not remove
+
+* **Video Filter:**
+
+** video/* 
+
+** .mp4 (MPEG-4 Part 14)
+** .avi (Audio Video Interleave)
+** .mov (Apple QuickTime Movie)
+** .wmv (Windows Media Video)
+** .flv (Flash Video)
+** .mkv (Matroska Video)
+** .webm (WebM Video)
+** .mpeg or .mpg (MPEG Video)
+** .vob (DVD Video Object)
+** .ogv (Ogg Video)
+** .m4v (MPEG-4 Video File)
+** .3gp (3GPP Multimedia File)
+** .3g2 (3GPP2 Multimedia File)
+** .asf (Advanced Systems Format)
+** .h264 (H.264 Encoded Video File)
+** .rm (RealMedia File)
+** .rmvb (RealMedia Variable Bitrate)
+** .ts (MPEG Transport Stream)
+** .mts (AVCHD Video File)
+** .m2ts (MPEG-2 Transport Stream)
+** .divx (DivX-Encoded Movie File)
+** .xvid (Xvid-Encoded Video File)
+** .dvr-ms (Microsoft Digital Video Recording)
+** .f4v (Flash MP4 Video File)
+** .m2v (MPEG-2 Video)
+** .mxf (Material Exchange Format)
+** .svi (Samsung Video File)
+** .m4p (MPEG-4 Protected File)
+** .qt (QuickTime Movie)
+** .nsv (Nullsoft Streaming Video File)
+** .amv (Anime Music Video File)
+** .flh (FLIC Animation File)
+** .roq (Id Software Game Video)
+** .mpe (MPEG Movie File)
+** .smk (Smacker Video File)
+** .bik (Bink Video File)
+** .ayuv (Uncompressed YUV Video File)
+
+// do not remove
+
+[none]
+** ... and all other types that are included within the MIME Types of video/*
+
+{empty}
+
+* **Audio Filter:**
+** audio/* +
+(e.g. **.mp3, .wav, .aac, .flac, .ogg, .m4a, .wma, .opus, .alac, .mid, .midi, .mp2, .amr, .aiff, .au, .ra, .ape, .ac3, .dts, .mka**)
+
+* **Archive Filter:**
```

Comment:
Audio file types are listed after this heading ???

Reply:
Fixed. This was a "hickup" because of a rebase conflict I solved but have oveseen. It is now correct. Thanks for catching 👍